qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -241,6 +259,11 @@ def wrapped_callback(found):
 
         self._widget.page().findText(text, flags, wrapped_callback)
 
+    def _on_find_finished(self, find_text_result):
```

Comment:
Adding a `@pyqtSlot` with the correct type (`QWebEngineFindTextResult`) isn't possible here because that type isn't available with `qt < 5.14`. Any suggestions on how to solve this?